### DISCUSSION TITLE
Change Hourglass Sand to uncommon (green) or rare (blue) so that it is available for group loot roll and master loot

### DISCUSSION THREAD

**Discussion by Santaria94** [chromiecraft/chromiecraft#3747] [Suggestions]
I suggest changing Hourglass Sand to uncommon or rare so that it is available for group loot roll and master loot. At the moment, a large amount of sand often remains in mobs, if the players who are assigned loot from the mob did not loot them. Changing the quality of the sand to Uncommon or Rare should help with this, as there will be more options in loot rules.

upd.

With the release of Zulgurub, there is now a similar problem with massive mojo. These items drop 10% from bosses and 100% from Hakkar, they are only needed to create an amulet that is needed to summon the boss. But these items are of normal quality and we can't get them with master loot, and random guys either don't loot bosses or refuse to give us massive mojos. In the end we have an alchemist with a recipe, we have all the necessary materials, except for these damn massive mojos, which we cannot reserve for the raid to summon the boss.

**Comment by plasma-93** [comment]
yes please

**Comment by balleny** [comment]
currently, I guess you can only go with free for all, if people arent looting anyway
